src/less/components/dark-mode.less

```diff
@@ -0,0 +1,458 @@
+@dark-mode-background-1: #19191B;
+@dark-mode-background-2: #1B1B20;
+@dark-mode-background-3: #1D1D22;
+@dark-mode-background-4: #22222A;
```

Comment:
I've seen this color at some places, it got a blue/purple tint to it, is this intended?
Tried out `#1F1F24` from the sidebar too but I don't know what I like better.

Reply:
yeah it's intentional, i might try tweaking the sidebar colors more in the future but looks good enough already